### ISSUE TITLE
[Bug] PiP is forced for some videos (even though the option the disabled in settings)

### ISSUE THREAD

**Issue by JayceTheBased** [Balackburn/YTLitePlus#62]
### Is there an existing issue for this?

- [x] I have searched the existing issues

### Have you read the FAQ?

- [X] Yes, I read the FAQ

### Device info

- iOS/iPadOS version: 15.8
- Device model: iPhone 7
- Sideload tool (AltStore, Sideloadly, TrollStore,...): Sideloadly
- The specific version of YTLitePlus (**latest** or **newest** is **NOT** a version number!): 18.49.3_2.4

### Describe the bug

Let me just start off by saying thank you so much for your hard work on this app. I really appreciate it.

So I often like to use YouTube to play music on my phone, sometimes while I'm browsing the web or making notes on my phone at the same time. I always have PiP turned off because I don't like it. However, I just noticed that although I have the PiP option disabled, the PiP still pops up for some videos only when I switch apps or go to the home screen. I'm not sure why it consistently pops up for some videos and not others, but I'll include a number of videos that I've noticed the problem with.

I did try enabling the legacy PiP option to see if that changed anything - it didn't seem to.

### Steps to reproduce the issue

Here are some videos this issue occurs with:

1. https://www.youtube.com/watch?v=6wPWFWUsPBw
2. https://www.youtube.com/watch?v=jiwuQ6UHMQg
3. https://www.youtube.com/watch?v=b58rgdK9dTI&list=PL-mmvz0wGt0DPr6-0qXZgPdyM2tgt63A5&index=134 
4. https://www.youtube.com/watch?v=nWm_OhIKms8

### Are you using the newest version of YTLitePlus? If not, why?

✅ Yes, I'm using the latest version of YTLitePlus right now

### Does the issue happen with the official YouTube from AppStore?

❌ No, It doesn't

**Comment by Balackburn** [comment]
Hello I think this isn't from YTLitePlus but from Youtube itself, some videos have pip randomly activated even without Premium idk why.

**Comment by JayceTheBased** [comment]
> Hello I think this isn't from YTLitePlus but from Youtube itself, some videos have pip randomly activated even without Premium idk why.

Thanks for your response and sorry my response has been really slow lol. You are absolutely correct - I figured out the problem and it's not anything specific to YTLitePlus. After thinking about it, I remembered that in the good old days of me having a Galaxy S8 (before I broke it, smh), I had the exact same problem with the Android app YouTube Revanced. To fix it, I just had to disable the PiP in the Android system settings for the YouTube app.

So to fix the problem with this app, I had to do something similar, specifically, I had to turn off "Start Picture in Picture automatically" in the general section of iOS settings. I guess that the issue has something to do with the way the background playback is emulated - probably because it's client side, not server side. 

In other words, I'm pretty sure that the reason why some videos have PiP activated is because they're actual videos (as opposed to pure audio streams with a static image background, which is what you get with a lot of music channels and the like). Anything with an actual video stream activates the PiP.

Anyway, by changing that setting I got rid of the PiP, so you can close this issue if you want.
